content/en/docs/tasks/traffic-management/egress/egress-tls-origination/index.md

```diff
@@ -194,17 +194,146 @@ are being sent by inspecting [Server Name Indication (SNI)](https://en.wikipedia
 The _SNI_ field is sent unencrypted during the TLS handshake. Using HTTPS prevents the attackers from knowing specific
 topics and articles but does not prevent an attackers from learning that `edition.cnn.com` is accessed.
 
-## Cleanup
+### Cleanup the TLS origination example
 
-1.  Remove the Istio configuration items you created:
+Remove the Istio configuration items you created:
 
     {{< text bash >}}
     $ kubectl delete serviceentry edition-cnn-com
     $ kubectl delete destinationrule edition-cnn-com
     {{< /text >}}
 
-1.  Shutdown the [sleep]({{< github_tree >}}/samples/sleep) service:
+## Mutual TLS origination for egress traffic
+
+This section describes how to configure a sidecar to perform TLS origination for an external service, this time using a
+service that requires mutual TLS. This needs the following steps, some of them are similar to the steps mentioned in
+[Egress Gateway TLS Origination](/docs/tasks/traffic-management/egress/egress-gateway-tls-origination)
```

Comment:
This example is considerably more involved because it requires the following setup: